modules/auxiliary/scanner/winrm/winrm_login.rb

```diff
@@ -35,15 +35,6 @@ module without SSL, the 'AllowUnencrypted' winrm option must be set.
     )
 
     deregister_options('PASSWORD_SPRAY')
-
```

Comment:
Removed this because it's redundant with the `include Msf::Exploit::Remote::WinRM` on L12.